qa/test-harness/full-test-sample.ini

```diff
@@ -0,0 +1,163 @@
+[_meta]
+PerformBundleIdCheck = True
+
+[test1_bsmLogDuringEvent]
```

Comment:
What's the significance of `test1_...`? Can't the `key` be the same thing as the DataFile path? I found that to be redundant for odevalidator config.ini and making the path the same as the key not only eliminated the redundancy, it also provided for the `key` to be unique naturally.

Reply:
Changed, key is now filepath.